provider/cmd/app/app.go

```diff
@@ -372,7 +391,7 @@ func runGRPCServer(ctx context.Context, setupLog logr.Logger, log logr.Logger, s
 
 	grpcSrv := grpc.NewServer(
 		grpc.ChainUnaryInterceptor(
-			commongrpc.InjectLogger(log),
+			commongrpc.InjectLogger(log.WithName("grpc-server")),
```

Comment:
optional: why grpc? it is iri protocol server, which implement your iri protocol. grpc is used technology.

Reply:
fixed